keymap.c

```diff
@@ -1579,15 +1603,15 @@ void mutt_what_key(void)
 {
   int ch;
 
-  mutt_window_mvprintw(MuttMessageWindow, 0, 0, _("Enter keys (^G to abort): "));
+  mutt_window_mvprintw(MuttMessageWindow, 0, 0, _("Enter keys (press abort key to resume): "));
```

Comment:
It'd be nice to tell the user _which_ key that is.
See: `km_find_func()`, `km_expand_key()` in `mutt_body_handler()`

Reply:
Good point, I've changed it inbe1b66b3f725cef6f31c5a9d1e23451781d4dfa0.